spec.md

```diff
@@ -56,16 +60,33 @@ elements it contains.
 We also use the following operators and functions:
 
 - $x \wedge y$ denotes the bitwise AND of $x$ and $y$
-- $x \vee y$ denotes the bitwise OR of $x$ and $y$
+- $x \vee y$ denotes the bitwise *inclusive* OR of $x$ and $y$
+- $x \oplus y$ denotes the bitwise *exclusive* OR of $x$ and $y$
 - $x \ll n$ denotes shifting $x$ to the left $n$ bits, i.e.
   $x \ll n = x2^{n}$
 - $x \gg n$ denotes a *logical* right shift -- it shifts $x$ to the
   right by $n$ bits, i.e. $x \gg n = x / 2^n$
-- $X \mathbin{\|} Y$ denotes the concatenation of two sequences $X$ and $Y$,
+- $X \mathbin{\|} Y$ denotes the concatenation of two sequences $X$ and
+  $Y$,
   i.e. if $X = \langle X_0, \dots, X_N \rangle$ and $Y = \langle Y_0,
   \dots, Y_M \rangle$ then $X \mathbin{\|} Y = \langle X_0, \dots, X_N, Y_0, \dots, Y_M
   \rangle$
 - $\operatorname{min}(x, y)$ denotes the minimum of $x$ and $y$.
+- $\operatorname{ROT}_L(x, n)$ denotes the rotation of $x$ to the left
+  by $n$ bits, i.e. $\operatorname{ROT}_L(x, n) = (x \ll n) \vee (x \gg
+  (32 - n))$
+
+We use standard mathematical notation for summation. For example:
+
+$\sum_{i = 0}^{n} i$
+
+denotes the sum of integers in the range $[0, n]$.
+
+We define a similar notation for exclusive or:
+
+$\bigoplus_{i = 0}^{n} i$
+
+denotes the bitwise exclusive or of theintegers in $[0, n]$.
```

Comment:
```suggestion
denotes the bitwise exclusive or of the integers in $[0, n]$.
```

Reply:
Also, the term "exclusive or" can't be applied to an arbitrary sequence of integers as readily as "sum" can. I think you need to specify that this means the first number in the sequence xor'd with each successive value in the sequence.

Reply:
Yeah, I guess we should be a bit more precise here.